src/templates/utils/index.ts

```diff
@@ -1,17 +1,20 @@
 import { Template } from '../template';
 
 interface ProjectArgs {
-  name: string;
+  safeName: string;
   author: string;
+  pkg: string;
 }
+
 export const composePackageJson = (template: Template) => ({
-  name,
+  safeName,
   author,
+  pkg,
 }: ProjectArgs) => {
   return {
     ...template.packageJson,
-    name,
+    name: pkg,
```

Comment:
```suggestion
    name,
```

Reply:
So removing `safeName` here will actually create potential for an error -- there's no check that what you typed in is a valid `package.json.name`. What we can do to workaround this is add a check in `create` and ask for a name again if the name is invalid, so only valid names make it here.